### PR TITLE
VMware VM display changes

### PR DESCRIPTION
**Pull request by Rosiak**

### CODE REVIEW

File: html/pages/device/vm.inc.php [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
+foreach (dbFetchRows('SELECT `vmwVmDisplayName`,`vmwVmState`,`vmwVmGuestOS`,`vmwVmMemSize`,`vmwVmCpus` FROM `vminfo` WHERE `device_id` = ? ORDER BY `vmwVmDisplayName', array($device['device_id'])) as $vm) {
     include 'includes/print-vm.inc.php';
-
     $i++;
 }
 
 echo '</table>';
-
-$pagetitle[] = 'Virtual Machines';
+$pagetitle[] = 'Virtual Machines';
```
Comment (paulgear) [inline html/pages/device/vm.inc.php]: What editor are you using?  Would be nice to get it not to mess up EOF newlines like that...

**Comment by paulgear** [comment]
:+1: from me

File: html/pages/device/vm.inc.php (hunk 1 above)
Comment (Rosiak) [inline html/pages/device/vm.inc.php]: Yeah sorry for that, should I re-add it again?

Comment (paulgear) [inline html/pages/device/vm.inc.php]: I would prefer it, but it's not a big deal; it just adds noise to diffs is all...

File: html/pages/device/vm.inc.php [hunk 2]
```diff
@@ -1,15 +1,11 @@
 <?php
-
-echo '<table border="0" cellspacing="0" cellpadding="5" width="100%" class="sortable"><tr class="tablehead"><th>Server Name</th><th>Power Status</th><th>Operating System</th><th>Memory</th><th>CPU</th></tr>';
-
+echo '<table border="0" cellspacing="0" cellpadding="5" width="100%" class="table table-condensed"><tr class="tablehead"><th>Server Name</th><th>Power Status</th><th>Operating System</th><th>Memory</th><th>CPU</th></tr>';
 $i = '1';
 
-foreach (dbFetchRows('SELECT * FROM vminfo WHERE device_id = ? ORDER BY vmwVmDisplayName', array($device['device_id'])) as $vm) {
+foreach (dbFetchRows('SELECT `vmwVmDisplayName`,`vmwVmState`,`vmwVmGuestOS`,`vmwVmMemSize`,`vmwVmCpus` FROM `vminfo` WHERE `device_id` = ? ORDER BY `vmwVmDisplayName', array($device['device_id'])) as $vm) {
```
Comment (f0o) [inline html/pages/device/vm.inc.php]: Missing ending 
```
` tick
```
In SQL

**Pull request merged**